src/components/BottomBar.vue

```diff
@@ -1,50 +1,87 @@
+      <div class="bottombar-tabviewTitleWrapper">
+        <div class="bottombar-tabview" v-bind:class="{ expandedTabView: isExpanded }">
+            <bottombartabview
+            :bottomCourses="bottomCourses"
+            :seeMoreCourses="seeMoreCourses"
+            :isExpanded="isExpanded"
+            @bottomBarTabToggle="bottomBarTabToggle"
+            @toggleFromTab="toggleFromTab"
+            />
+        </div>
+        <div class="bottombar-title" @click="toggle()">
+          <bottombartitle
+          :color="bottomCourses[0].color"
+          :name="bottomCourses[0].name"
+          :isExpanded="isExpanded"
+          />
+        </div>
+      </div>
+      <div v-if="this.isExpanded" class="bottombar-course">
         <bottombarcourse
-        :subject="data.subject"
-        :number="data.number"
-        :name="data.name"
-        :credits="data.credits"
-        :semesters="data.semesters"
-        :color="data.color"
-        :id="data.id"
-        :instructors="data.instructors"
-        :distributionCategories="data.distributionCategories"
-        :enrollmentInfo="data.enrollmentInfo"
-        :latestSem="data.latestSem"
-        :latestLecInfo="data.latestLecInfo"
-        :overallRating="data.overallRating"
-        :difficulty="data.difficulty"
-        :workload="data.workload"
-        :prerequisites="data.prerequisites"
-        :description="data.description"
-        :isPreview="data.isPreview"
-        :isExpanded="data.isExpanded"
-        @toggle="toggle"
+        :courseObj="bottomCourses[0]"
         />
+      </div>
     </div>
 </template>
 
 
 <script>
 import Vue from 'vue';
 import BottomBarCourse from '@/components/BottomBarCourse';
+import BottomBarTabView from '@/components/BottomBarTabView';
+import BottomBarTitle from '@/components/BottomBarTitle';
 
 Vue.component('bottombarcourse', BottomBarCourse);
+Vue.component('bottombartabview', BottomBarTabView);
+Vue.component('bottombartitle', BottomBarTitle);
+
 export default {
   props: {
     bottomCourses: Array,
-    data: Object
+    seeMoreCourses: Array,
+    isExpanded: Boolean
   },
+
   methods: {
-    toggle(isExpanded) {
-      if (isExpanded) this.$emit('close-bar');
+    toggle() {
+      if (this.isExpanded) this.$emit('close-bar');
       else this.$emit('open-bar');
+    },
+    bottomBarTabToggle(courseObj) {
+      // Move courseObj to front of array
+      if (this.bottomCourses.indexOf(courseObj) > 0) { // not already in front
+        this.bottomCourses.splice(this.bottomCourses.indexOf(courseObj), 1);
+        this.bottomCourses.unshift(courseObj);
+      }
+    },
+    toggleFromTab() {
+      this.toggle();
     }
   }
 };
+
 </script>
 
 
 <style scoped lang="scss">
+.bottombar {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+
+  &-tabview {
+    position:fixed;
+    bottom: 2.5rem;
+    left: 29.5rem;
+    width: calc(100vw - 29.5rem);
+  }
+
+}
+.expandedTabView{
+    position: fixed;
+    bottom: 18.81rem;
```

Comment:
Can this be rounded to something better?

Reply:
I used an even pixel size and reconverted to rem and got 18.75rem. Is that ok?

Reply:
This is for the See More screenshot but I noticed that the scrollbar might cover a part of the X on hover if a user wants to delete a See More course. Should I move the X more inward?

Reply:
This all sounds good to me, thanks Theresa